### ISSUE TITLE
Tooltip configuration freezes the page

### ISSUE THREAD

**Issue by GeryStoycheva** [apache/echarts#15209]
### Version
5.1.1

### Steps to reproduce
When you have grid with two or more charts and add tooltips to the configuration, the whole page freezes and the CPU usage gets high.

Example configuration:
```
option = {
    grid: [
        {
            top: 10,
            bottom: 0,
            height: 200,
            containLabel: true,
        },
        {
            top: 250,
            bottom: 0,
            height: 200,
            containLabel: true,
        }
    ],
	tooltip: [
		{
			trigger: "axis"
		},
		{
			trigger: "axis"
		}
	],
	axisPointer: {
		link: { xAxisIndex: "all" },
	},
    xAxis: [
        {
            type: 'time',
            gridIndex: 0,
        },
         {
            type: 'category',
            gridIndex: 0,
            data: ['Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat', 'Sun']
        },
        {
            type: 'time',
            gridIndex: 1,
        },
         {
            type: 'category',
            gridIndex: 1,
            data: ['Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat', 'Sun']
        },
    ],
    yAxis: [
        {
            type: 'value',
            gridIndex: 0
        },
        {
            type: 'value',
            gridIndex: 1
        }
    ],
    series: [
        {
            name: "1-1",
            data: [["1988/10/4", 26], ["1988/10/5", 25], ["1988/10/6", 34], ["1988/10/7", 41], ["1988/10/8", 45]],
            type: 'line',
            xAxisIndex: 0,
            yAxisIndex: 0,
        },
        {
            name: "1-2",
            data: [["1988/10/4", 35], ["1988/10/5", 40], ["1988/10/6", 17], ["1988/10/7", 10], ["1988/10/8", 45]],
            type: 'line',
            xAxisIndex: 0,
            yAxisIndex: 0,
        },
        {
            name: "2-1",
            data: [["1988/10/4", 26], ["1988/10/5", 25], ["1988/10/6", 34], ["1988/10/7", 41], ["1988/10/8", 45]],
            type: 'line',
            xAxisIndex: 2,
            yAxisIndex: 1,
        },
        {
            name: "2-2",
            data: [["1988/10/4", 35], ["1988/10/5", 40], ["1988/10/6", 17], ["1988/10/7", 10], ["1988/10/8", 45]],
            type: 'line',
            xAxisIndex: 2,
            yAxisIndex: 1,
        },
    ]
};
```

**CPU usage:**
![Screenshot_6](https://user-images.githubusercontent.com/86286609/122881970-61fadf00-d344-11eb-8ee3-484da5496a89.png)

### What is expected?
Page not freezing.

### What is actually happening?
Page freezes.

**Comment by pissang** [comment]
Only one tooltip component is supported currently

**Comment by bozhidarc** [comment]
+1
Same problem here.

**Comment by pissang** [comment]
Fixed in 5.3
